web/assets/main.js

```diff
@@ -1,10 +1,10 @@
-import {AwsRum} from 'aws-rum-web';
-import * as MOJFrontend from '@ministryofjustice/frontend'
+import * as MOJFrontend from '@ministryofjustice/frontend';
+import { AwsRum } from 'aws-rum-web';
```

Comment:
We're playing ping pong with curly brace styling - is there a style config this is being picked up from? I think it's pre-commit that reverts it, so can look at making sure we're pointing to the same config.